control/iosys.py

```diff
@@ -369,23 +369,24 @@ def _rhs(self, t, x, u, params={}):
         NotImplemented("Evaluation not implemented for system of type ",
                        type(self))
 
-    def dynamics(self, t, x, u):
+    def dynamics(self, t, x, u, params={}):
```

Comment:
Is there a reason not to use the `params=None`, `if params is None: params={}` pattern here?  Using mutable objects for default parameters is usually not a good idea.

Also, why not **kwargs?